cdk/tsconfig.eslint.json

```diff
@@ -0,0 +1,5 @@
+{
+  "extends": "./tsconfig.json",
+  "include": ["lib/**/*", "bin/**/*"],
+  "exclude": ["node_modules", "../hq/public/javascripts"]
```

Comment:
😨  Does this mean ESLint looks at the entire repo and not the directory it's installed in?!

Reply:
apparently so

Reply:
Although I'm not sure why! I'll check this.

Reply:
Okay, can remove this. I think I must have been linting the wrong directory at the time.